addons/udes_common/models/models.py

```diff
@@ -105,3 +105,59 @@ def selection_display_name(self, selection_field_name):
     # Odoo has a nice function that handles all the weird ways you can define selections!
     selection_values_dict = dict(selection_field._description_selection(self.env))
     return selection_values_dict.get(self[selection_field_name], False)
+
+
+# N.B: Basic values should not have any relations as this is to stop the recursion
```

Comment:
Will change the comment later , Basic values are not to stop the recursion , max_level stops the recursion